explorer/src/test/java/com/scalar/client/tool/explorer/ExplorerTest.java

```diff
@@ -24,27 +24,28 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
-
-import com.scalar.client.config.ClientConfig;
-import com.scalar.client.service.ClientService;
-import com.scalar.client.service.StatusCode;
-import com.scalar.rpc.ContractExecutionResponse;
-import com.scalar.rpc.LedgerServiceResponse;
-import com.scalar.rpc.LedgerValidationResponse;
+import static org.mockito.Mockito.*;
+
+import com.scalar.dl.client.config.ClientConfig;
+import com.scalar.dl.client.exception.ClientException;
+import com.scalar.dl.client.service.ClientService;
+import com.scalar.dl.ledger.model.ContractExecutionResult;
+import com.scalar.dl.ledger.model.LedgerValidationResult;
+import com.scalar.dl.ledger.service.StatusCode;
+import com.scalar.dl.rpc.ContractExecutionResponse;
+
+import java.io.StringReader;
 import java.util.Optional;
 import javax.json.Json;
 import javax.json.JsonArray;
 import javax.json.JsonObject;
+
```

Comment:
Did you apply google format ?

Reply:
Thank you for review comment.
Code modified with google format